pkg/controller/scyllacluster/resource/resource.go

```diff
@@ -119,6 +119,14 @@ func memberServicePorts(cluster *scyllav1.ScyllaCluster) []corev1.ServicePort {
 			Name: "cql-ssl",
 			Port: 9142,
 		},
+		{
+			Name: "cql-shard-aware",
+			Port: 19042,
+		},
+		{
+			Name: "cql-shard-aware-ssl",
```

Comment:
nit: if the original port is named `cql-ssl` would it be better to use `cql-ssl-shard-aware`?

Reply:
done